src/blas/backends/rocblas/rocblas_handle.hpp

```diff
@@ -0,0 +1,62 @@
+/***************************************************************************
+*  Copyright (C) Codeplay Software Limited
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  For your convenience, a copy of the License has been included in this
+*  repository.
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+**************************************************************************/
+#ifndef RBLAS_HANDLE_HPP
+#define RBLAS_HANDLE_HPP
+#include <atomic>
+#include <unordered_map>
+#include "rocblas_helper.hpp"
+
+namespace oneapi {
+namespace mkl {
+namespace blas {
+namespace rocblas {
+
+template <typename T>
+struct rblas_handle {
+    using handle_container_t = std::unordered_map<T, std::atomic<rocblas_handle> *>;
+    handle_container_t rblas_handle_mapper_{};
```

Comment:
same here: `rocblas_handle_mapper_`

Reply:
We'll address this in the next PR

Reply:
@TejaX-Alaghari  This change is simple enough that, it does not require a separate PR. Could you please update the current PR?